### ISSUE TITLE
Fix build removing usage of itkMultiplyByConstantImageFilter

### ISSUE THREAD

**Issue by jcfr** [ljzhu/LASegmenter#4]
Fix build error by updating code to use `itkDisplacementFieldJacobianDeterminantFilter` instead of `itkMultiplyByConstantImageFilter`.

The filter `itkMultiplyByConstantImageFilter` is deprecated and available only when building ITK with `ITKV3_COMPATIBILITY` set to 1 and this is **NOT** the case anymore in Slicer.

```
[ 50%] Building CXX object CLILASegmenter/CMakeFiles/LASegmenterLib.dir/LASegmenter.cxx.o
In file included from /tmp/LASegmenter/CLILASegmenter/LASegmenter.cxx:9:0:
/tmp/LASegmenter/CLILASegmenter/lib/utilities.hxx:30:46: fatal error: itkMultiplyByConstantImageFilter.h: No such file or directory
compilation terminated.
CLILASegmenter/CMakeFiles/LASegmenterLib.dir/build.make:68: recipe for target 'CLILASegmenter/CMakeFiles/LASegmenterLib.dir/LASegmenter.cxx.o' failed
make[2]: *** [CLILASegmenter/CMakeFiles/LASegmenterLib.dir/LASegmenter.cxx.o] Error 1
CMakeFiles/Makefile2:1089: recipe for target 'CLILASegmenter/CMakeFiles/LASegmenterLib.dir/all' failed
make[1]: *** [CLILASegmenter/CMakeFiles/LASegmenterLib.dir/all] Error 2
Makefile:162: recipe for target 'all' failed
make: *** [all] Error 2
```

Error similar to
* https://github.com/tokjun/ErodeDilateLabel/issues/2
* https://github.com/gaoyi/SegmentationAidedRegistration/issues/7

**Comment by jcfr** [comment]
Good news. This was fixed for `LASegmenter` by @blowekamp. See https://github.com/ljzhu/LASegmenter/pull/3

**Comment by ljzhu** [comment]
@jcfr Hi JC, I am a bit confused. Based on the comment on Aug 31, it says the the issue was fixed. See #3. Could you let me know what else needs to be integrated? Thanks!

**Comment by jcfr** [comment]
It is indeed mentioned that it was fixed in the pull request #3 not in the master branch of your project. The fix will effectively be integrated by merging the changes instead of just closing the pull request. 

For your convenience, I just rebased the PR#3 and created a new one.

To summarize, consider merging https://github.com/ljzhu/LASegmenter/pull/5 and https://github.com/ljzhu/LASegmenter/pull/6

**Comment by jcfr** [comment]
Well done 👍 

Closing. Fix integrated in https://github.com/ljzhu/LASegmenter/commit/f5837c18e375eb052550963a08a0a3c6fdae1ad8
